cfpb/consumerfinance.gov
# Add Django-Watchman check for Elasticsearch health

This PR adds a check for Elasticsearch cluster health to our Django-Watchman configuration. It should report "ok" unless the Elasticsearch API reports a time-out or if the cluster status isn't "green". It includes the ES health API output at the shard-level.

This PR also removes the Django-Watchman configuration we have that we haven't been using. It also updates the default Watchman token generation to use the Python 3 `secrets` library.

Locally, [`WATCHMAN_TOKENS` is empty](https://github.com/cfpb/consumerfinance.gov/blob/main/.env_SAMPLE#L176) so Watchman should be available http://localhost:8000/_status/

## How to test this PR

1. Visit http://localhost:8000/_status/ and observe the Elasticsearch check output.

## Checklist
